content/zh/docs/hertz/tutorials/service-governance/service_discovery.md

```diff
@@ -1943,3 +1994,469 @@ func main() {
 ### 完整示例
 
 完整用法示例详见 [example](https://github.com/hertz-contrib/registry/tree/main/zookeeper/example) 。
+
+## Redis
+
+### 安装
+
+```go
+go get github.com/hertz-contrib/registry/redis
+```
+
+### 服务注册
+
+#### Option
+
+Redis 拓展在服务注册部分中提供了 option 配置。
```

Comment:
Add a table for configuration list.

Reply:
> Add a table for configuration list.

The table is not provided here to match the previous document.